include/volk/volk_common.h

```diff
@@ -5,84 +5,86 @@
 // Cross-platform attribute macros
 ////////////////////////////////////////////////////////////////////////
 #if _MSC_VER
-#  define __VOLK_ATTR_ALIGNED(x) __declspec(align(x))
-#  define __VOLK_ATTR_UNUSED
-#  define __VOLK_ATTR_INLINE     __forceinline
-#  define __VOLK_ATTR_DEPRECATED __declspec(deprecated)
-#  define __VOLK_ATTR_EXPORT     __declspec(dllexport)
-#  define __VOLK_ATTR_IMPORT     __declspec(dllimport)
-#  define __VOLK_PREFETCH(addr)
-#  define __VOLK_ASM __asm
-#  define __VOLK_VOLATILE
+#define __VOLK_ATTR_ALIGNED(x) __declspec(align(x))
```

Comment:
did you run clang-format?

Reply:
I don't mind; but please make it a separate commit, so that we can easily review the critical changes.